DuckDuckGo/AppDelegate+AppDeepLinks.swift

```diff
@@ -43,12 +50,21 @@ extension AppDelegate {
 
         case .fireButton:
             mainViewController.forgetAllWithAnimation()
+            if url.getParameter(named: "ls") != nil {
+                Pixel.fire(pixel: .lockScreenWidgetFireButton)
+            }
 
         case .voiceSearch:
             mainViewController.onVoiceSearchPressed()
+            if url.getParameter(named: "ls") != nil {
+                Pixel.fire(pixel: .lockScreenWidgetVoiceSearch)
+            }
 
         case .newEmail:
             mainViewController.newEmailAddress()
+            if url.getParameter(named: "ls") != nil {
+                Pixel.fire(pixel: .lockScreenWidgetNewEmail)
+            }
```

Comment:
perhaps we could make a nested function inside handleAppDeepLink
func firePixelIfNeeded(_ pixel: PixelEvent) {} so there is just 1 liner instead of 3 lines duplicated 5 times